src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt

```diff
@@ -149,19 +150,7 @@ object AntiAFK : Module() {
         TURN(turn)
     }
 
-    private val isBaritoneActive: Boolean = if (mc.player != null) BaritoneAPI.getProvider().primaryBaritone.customGoalProcess.isActive else false
-
     private fun baritoneGotoXZ(x: Int, z: Int) {
         BaritoneAPI.getProvider().primaryBaritone.customGoalProcess.setGoalAndPath(GoalXZ(x, z))
     }
-
-    private fun baritoneCancel() {
-        if (walk.value && isBaritoneActive) BaritoneAPI.getProvider().primaryBaritone.pathingBehavior.cancelEverything()
-    }
-
-    init {
-        walk.settingListener = Setting.SettingListeners {
-            if (isBaritoneActive) BaritoneAPI.getProvider().primaryBaritone.pathingBehavior.cancelEverything()
-        }
-    }
```

Comment:
Revert this, it's not "useless"

Reply:
What exactly does this do? I didn't observe any difference with it in and out.

Reply:
![image](https://user-images.githubusercontent.com/17222512/97246289-5ebe4c00-17d3-11eb-81ec-1da3cddfc0c9.png)

Reply:
Done. thanks.